R/html-tweak.R

```diff
@@ -108,7 +112,20 @@ tweak_navbar_links <- function(html, pkg = pkg) {
     )
   }
 
-  return(as.character(xml2::xml_find_first(html, ".//body")))
+  scripts <- xml2::xml_find_all(html, ".//script")
+  scripts <- scripts[!is.na(xml2::xml_attr(scripts, "src"))]
+  srcs <- xml2::xml_attr(scripts, "src")
+
+  needs_tweak <- !grepl("https?\\:\\/\\/", srcs)
```

Comment:
Probably time to pull this out into it's own helper function.

Reply:
in the end with the refactoring I didn't need a helper function :thinking: